### DISCUSSION TITLE
How to add data from CSV file into Google Sheet using google_sheets4_cli?

### DISCUSSION THREAD

**Discussion by vitalyk-multinarity** [Byron/google-apis-rs#420] [Q&A]
I went over http://byron.github.io/google-apis-rs/google_sheets4_cli/spreadsheets_values-append/, but couldn't ask an answer to a simple question - how I can add a new row to an existing google sheet from CSV file (the same fields and number of fields).
TIA, Vitaly

**Comment by Byron** [comment]
Even though this seems like the natural place to come with questions on how to use the CLI, the answer is: it depends, very much depending on the API in question. Sometimes certain things don't work at all as the API isn't used the way as the designers intended, and it's impossible to know what it is.

Generally a good way for going about this is to find an example made by google, then translate this to the API crate and the learn how to do that in the CLI. Otherwise, experimentation with the CLI is the only alternative as generally it's able to drive the API crate in full.

I really wish Google would support Rust natively one day.
